### PR TITLE
Added some shell artifacts and moved shared unix shell related artifacts...

### PR DESCRIPTION
**Pull request by pidydx**
... to a combined location

### CODE REVIEW

File: definitions/shells.yaml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+      - '/root/.profile'
+      - '/root/.tcsh'
+      - '/root/.zlogin'
+      - '/root/.zlogout'
+      - '/root/.zprofile'
+labels: [Configuration Files]
+supported_os: [Linux]
+---
+name: UsersShellConfigs
+doc: Linux root shell configuration files.
```
Comment (destijl) [inline definitions/shells.yaml]: user

File: definitions/shells.yaml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    path_list:
+      - '/root/.bash_history'
+      - '/root/.sh_history'
+      - '/root/.zhistory'
+      - '/root/.zsh_history'
+labels: [Configuration Files]
+supported_os: [Linux]
+---
+name: UsersShellHistory
+doc: Linux root shell configuration files.
```
Comment (destijl) [inline definitions/shells.yaml]: user

File: definitions/shells.yaml [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+supported_os: [Linux, Darwin]
+---
+name: UnixPasswd
+doc: Linux or OSX passwd file.
+collectors:
+- collector_type: FILE
+  args: {path_list: ['/etc/passwd']}
+labels: [Authentication]
+supported_os: [Linux, Darwin]
+---
```
Comment (destijl) [inline definitions/shells.yaml]: this extra separator is probably causing the test failure.

File: definitions/shells.yaml [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+      - '/etc/zsh/zlogout'
+      - '/etc/zsh/zprofile'
+      - '/etc/zsh/zshenv'
+      - '/etc/zsh/zshrc'
+      - '/etc/zshenv'
+      - '/etc/zshrc'      
+labels: [Configuration Files]
+supported_os: [Linux, Darwin]
+---
+name: SudoersConfiguration
```
Comment (destijl) [inline definitions/shells.yaml]: these need to be removed from linux.yaml.  I feel like they should be in a separate file that covers the commonalities between os x and linux.  Maybe nix.yaml?

**Comment by pidydx** [comment]
I think I got all the cleanup things you mentioned.  I went with unix_common.yaml for the shared artifacts.

File: definitions/unix_common.yaml [hunk 5]
```diff
@@ -0,0 +1,111 @@
+# Common Unix based artifacts
```
Comment (destijl) [inline definitions/unix_common.yaml]: I think this file should only be those things that are shared between linux and OS X.  Move the linux-only ones back into the linux file.

File: definitions/unix_common.yaml [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+      - '/root/.profile'
+      - '/root/.tcsh'
+      - '/root/.zlogin'
+      - '/root/.zlogout'
+      - '/root/.zprofile'
+labels: [Configuration Files]
+supported_os: [Linux]
+---
+name: UsersShellConfigs
+doc: Common unix user shell configuration files.
```
Comment (destijl) [inline definitions/unix_common.yaml]: And I think you can drop "Common"

**Pull request merged**